README.md

```diff
@@ -15,8 +15,9 @@ A way to A/B test config, or slowly turn on config for certain audiences or % of
 
 Branches
 --------
-
-* For all supported Moodle versions use the master branch.
+| Moodle version    | Branch           | PHP  |
+|-------------------|------------------|------|
+| Moodle 3.8+       | MOODLE_38_STABLE | 7.1+ |
```

Comment:
Just to confirm we are no longer supporting 3.4 - 3.7 as stated in the documentation further below? If so it might be good to keep master as the version which supports the older Moodle versions, and also add this information to the table in the branches section. 

However, if we are supporting 3.4 - 3.7 the table needs to be updated to reflect this and it would be nice to include some patch files for the backports, similar to the way objectfs has set it up here https://github.com/catalyst/moodle-tool_objectfs/tree/master/patch

Reply:
Whatever we decide here, I'm happy for the patch files and older version support to be added in a separate PR if it might take some time to do.

Reply:
Well, yeah, 3.4-3.7 should be added to the table. Added it with a new column `Backport`.
Having a patch which can be used to extend github actions down to 3.4 is a great idea. Will create a separate issue for this

Reply:
@keevan patch issue: #43